README.md

```diff
@@ -37,7 +37,7 @@ The last two points together make it very easy to build functions as sequences o
 
 
 
-Also see [Why Ramda?](http://fr.umio.us/why-ramda/), [Why Curry Helps](https://web.archive.org/web/20140714014530/http://hughfdjackson.com/javascript/why-curry-helps) and [Hey Underscore, You're Doing It Wrong!](https://www.youtube.com/watch?v=m3svKOdZijA&app=desktop).
+Also see [Why Ramda?](http://fr.umio.us/why-ramda/), [Why Curry Helps](https://web.archive.org/web/20140714014530/http://hughfdjackson.com/javascript/why-curry-helps), [Hey Underscore, You're Doing It Wrong!](https://www.youtube.com/watch?v=m3svKOdZijA&app=desktop) and comprehensive [Thinking in Ramda](http://randycoulman.com/blog/categories/thinking-in-ramda).
```

Comment:
i don't think "comprehensive" is required. Also, now that we have four in the list, maybe we should have a bulleted list instead?